### PR TITLE
Update README.md with more WM2r recent issues

### PR DESCRIPTION
**Pull request by Metadorius**
as per Discord convo here https://discord.com/channels/243411108940087297/802730777443958824/1137014426357223528

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,8 +19,18 @@ A list of things useful to owners of the GPD Win Max 2
 14. Make sure Dpad is working correctly.
 15. Make sure remote play is working correctly. (A user was having issues with the screen randomly going black and having to force the machine to shut down.)
 ###### Extra things to check for the Win Max 2 2023 refresh.
-1. Functionality of the onboard HDMI port.
-2. If the device is able to run stably with the RAM speed at 7500MT/s
+1. Functionality of the onboard HDMI port (some units arrived with them dead).
+2. If the device is able to run stably with the RAM speed at 7500MT/s.
+3. Check Advanced->AMD PBS->Display Configurations, make sure all is recognized properly.
+4. USB-C display out on both ports (note: hard to diagnose since USB-carried protocols are a mess)
```
Comment (Sabrina-Fox) [inline README.md:25]: move to section above, known issue on 2022 model as well so should add it to the section above

File: README.md [hunk 2]
```diff
@@ -19,8 +19,18 @@ A list of things useful to owners of the GPD Win Max 2
...
+5. Display having no retention, flickering or being halved.
```
Comment (Sabrina-Fox) [inline README.md:26]: move to section above, known issue on very few 2022 model as well so should add it to the section above

File: README.md [hunk 3]
```diff
@@ -19,8 +19,18 @@ A list of things useful to owners of the GPD Win Max 2
...
+6. M.2 2230 SSD and slot not being dead.
+7. Keyboard not being squeaky or otherwise weird.
```
Comment (Sabrina-Fox) [inline README.md:28]: not needed, inconsistent spacebars is due to the design and is already addressed in the known issue and workaround section

File: README.md [hunk 4]
```diff
@@ -19,8 +19,18 @@ A list of things useful to owners of the GPD Win Max 2
...
+8. Cooling fan may not be plugged in (happened on one unit).
+9. SD slots working as intended individually (note: they are not working simultaneously by design).
+10. Paint not flaking on gamepad covers.
```
Comment (Sabrina-Fox) [inline README.md:31]: move to section above, keyboard and gamepad layout is same as 2022 models

File: README.md [hunk 5]
```diff
@@ -19,8 +19,18 @@ A list of things useful to owners of the GPD Win Max 2
...
+11. Properly seated/fixed/installed shoulder controls
+12. Temps should be also checked, might need repasting
```
Comment (Sabrina-Fox) [inline README.md:33]: will remove for now until verified

**Review by Sabrina-Fox — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Metadorius) [reply README.md:28]: I don't think it's the same issue, as in it's not spacebar only and it's a different kind of issue, at least that's what I understood from the conversations around.

**Pull request closed without merging**